emily/handler/src/api/handlers/withdrawal.rs

```diff
@@ -94,6 +94,13 @@ pub async fn get_withdrawals(
             query.page_size,
         )
         .await?;
+        entries.iter().for_each(|entry| {
```

Comment:
Do we risk spamming the logs with this (and below)?

Reply:
Well, I suspect _usually_ it will be not so much withdrawals for recipient, so it will be around a few logs per one request to emily, which sounds legitimate for me. However, I don't mind to open an issue to remove this logs after I finish my debugging. But in general I'd say `debug!` log group is usually spammy one, so it is okish.

Reply:
I'm fine with having it for now for debugging, but we should probably reduce this to `trace!` later.